backend/internal/events/repo_test.go

```
@@ -129,3 +133,153 @@ func TestRepo_Search_EmptyTable(t *testing.T) {
+
+	return names
+}
+
+// seedFleet inserts vehicles numbered <prefix>-001.. each carrying crewPer
+// crew members.
+func seedFleet(t *testing.T, db *sql.DB, eventID, prefix string, vehicles, crewPer int) {
+	t.Helper()
+
+	for i := 1; i <= vehicles; i++ {
+		vehicleID := store.NewID()
+		_, err := db.Exec(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)